ui/src/app/pages/appAnalysis/app-selector.tsx

```diff
@@ -72,7 +72,12 @@ export const AppSelector = (props: AppSelectorProps) => {
         break;
       case UIAppType.SAS:
         setVisibleModal(VisibleModal.None);
-        openSASOrConfigPanel(workspace.namespace, userApps);
+        openSASOrConfigPanel(
```

Comment:
we can collapse the RStudio and SAS cases if we make this call fully generic.  that means the check can simply be `if (jupyter) doJupyter else doUserApp`

Reply:
Yes, there are other places as well that we can make generic. I would like to discuss with you once you are back and have a PR to do that. 
I am thinking apart from this file we could also generalize  RStudioButtonRow/SasButtonRow etc